platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBean.java

```diff
@@ -347,12 +366,14 @@ public void removeApplication(String appName) {
         }
     }
 
-    /** {@inheritDoc}.*/
+    /**
```

Comment:
We can delete those JavaDoc comments. You don't need to use `{@inheritDoc}` if you don't have any JavaDoc comments on your overriding method.